securitycenter/synth.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+import synthtool as s
+from synthtool import gcp
+
+gapic = gcp.GAPICGenerator()
+
+library = gapic.py_library(
+    'securitycenter',
+    'v1beta1')
+
+s.move(library, excludes=['setup.py', 'nox.py'])
```

Comment:
needs to exclude `nox*.py` (in case the generator begins emitting `noxfile.py`) along with `README.rst`, `docs/index.rst`, and `docs/conf.py`.

Reply:
```suggestion
excludes = [
    'setup.py',
    'nox*.py',
    'README.rst',
    'docs/index.rst',
    'docs/conf.py',
]
s.move(library, excludes=excludes)
```

Reply:
I don't think blanket excluding things make sense. I am expanding this after edits though
README.rst -> adding of badges
docs/index.rst -> DRYing basing on readme.

Is there a reason to, before altering it beyond generation, to exclude conf.py, nox*.py?

Reply:
- You've already modified `noxfile.py` -- we also don't want future synths to create `nox.py`.
- Re `docs/conf.py`:  the generated version is quite likely to screw up the package names (see `asset/docs/conf.py`'s git log).  We aren't actually building from those configs until after repo split, but once that happens, I'm definitely not going to want the generator touching it.

Reply:
RE: nox. nox.py is exlcuded, but why would we exclude noxfile.py if we started generating it? It would only show up if we added that support and I would think that would be the time to block it if needed. I expect in short time though noxfile.py will be from our templating system and we won't even bother with the generator for these files in the future.

RE: conf.py that makes sense enough. The file isn't particularly interesting. I can exclude this one to avoid random-ish failures.